src/clients/cli.rs

```diff
@@ -2,7 +2,7 @@ use crate::{
     core::{env, env::GetEnvValue, logs::LogStream, ports::Ports, ContainerState, Docker, WaitFor},
     Container, Image, ImageArgs, RunnableImage,
 };
-use shiplift::rep::ContainerDetails;
+use bollard::models::ContainerInspectResponse;
```

Comment:
If we would inline this struct, then we could feature-flag `bollard` on the `experimental`. What do you think?

Reply:
Interesting idea. Not sure about inlining the struct, though — we'd have to track the changes in the API, and it's not fun :) We could use [bollard-stubs](https://crates.io/crates/bollard-stubs) (it's the crate where bollard stores the autogenerated models) inside CLI; it should be enough. Are you OK with it?

Reply:
Yep, that is much better! Certainly more lightweight!